bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ChannelItemProvider.java

```diff
@@ -265,6 +301,29 @@ private void createItemForLink(ItemChannelLink link) {
         }
     }
 
+    private Collection<String> getItemType(Channel channel) {
```

Comment:
Can we change the method name to something that state what the implementation is doing (and perhaps we should use plural)?
For example `getProfileItemTypes` or something similar.
Sure, it is a private method, but it should be clear that the item types depend on the profiles and not on the accepted item type of the channel (without reading the implementation).